src/main/java/az/edu/turing/modul1/Task2B.java

```diff
@@ -0,0 +1,68 @@
+package az.edu.turing.modul1;
+
+import java.util.Scanner;
+
+public class Task2B {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("---------Welcome to the Week's planner----------");
+        String[][] weeks = new String[7][2];
+        weeks[0][0] = "Sunday";
+        weeks[0][1] = "do home Work!";
+        weeks[1][0] = "Monday";
+        weeks[1][1] = "learning English ";
+        weeks[2][0] = "Tuesday";
+        weeks[2][1] = "watching film";
+        weeks[3][0] = "Wednesday";
+        weeks[3][1] = "work in DataStructures";
+        weeks[4][0] = "Thursday";
+        weeks[4][1] = "have a fun with friends";
+        weeks[5][0] = "Friday";
+        weeks[5][1] = "go to the fitness";
+        weeks[6][0] = "Saturday";
+        weeks[6][1] = "run in the morning!";
+
+        while (true) {
+            System.out.println("""
+                    Choose the Menu:
+                    1.See today's plan
+                    2.Change plan of the day
+                    3.Exit""");
```

Comment:
try to solve without menu, get input as a string and then take action based on this

Reply:
I always try to write with this method of the menu,beacuse We will replace the int value with a string, what does it mean for us?

Reply:
Try to write based on conditions please. get input as a string and then take action based on this